hal/src/sercom/pad/impl_pad_thumbv7em.rs

```diff
@@ -429,28 +429,28 @@ pad_table!(
     PB26 {
         #[hal_cfg("sercom2")]
         C: (Sercom2, Pad0, IoSet2),
-        #[hal_cfg("sercom4")]
+        #[hal_cfg(all("sercom4", not("hsm")))]
         D: (Sercom4, Pad1, IoSet4),
     }
     #[hal_cfg("pb27")]
     PB27 {
         #[hal_cfg("sercom2")]
         C: (Sercom2, Pad1, IoSet2),
-        #[hal_cfg("sercom4")]
+        #[hal_cfg(all("sercom4", not("hsm")))]
         D: (Sercom4, Pad0, IoSet4),
     }
     #[hal_cfg("pb28")]
     PB28 {
         #[hal_cfg("sercom2")]
         C: (Sercom2, Pad2, IoSet2),
-        #[hal_cfg("sercom4")]
+        #[hal_cfg(all("sercom4", not("hsm")))]
         D: (Sercom4, Pad2, IoSet4),
     }
     #[hal_cfg("pb29")]
     PB29 {
         #[hal_cfg("sercom2")]
         C: (Sercom2, Pad3, IoSet2),
-        #[hal_cfg("sercom4")]
+        #[hal_cfg(all("sercom4", not("hsm")))]
```

Comment:
It's not immediately clear to me what this code actually means - if the HSM is present, does the chip have SERCOM4?

Reply:
Chips which contain the HSM module have these IOs hard wired to sercom 2 , thus cannot be used by any other peripheral (and the IOs are not even connected to the legs of the chip, so are not externally exposed)